src/types/Observations.ts

```diff
@@ -56,7 +56,8 @@ export type ObservationMonitoringPlotPhoto = components['schemas']['ObservationM
 
 // species related observation statistics
 export type ObservationSpeciesResultsPayload = components['schemas']['ObservationSpeciesResultsPayload'];
-export type ObservationSpeciesResults = ObservationSpeciesResultsPayload & {
+export type ObservationSpeciesResults = Omit<ObservationSpeciesResultsPayload, 'moralityRate'> & {
```

Comment:
thanks, this is great... sorry for the flip-flopping.. should we maybe show '--' for null mortality rates? maybe 0 is misleading.. don't know. we could check with Steve/design team maybe? Once again, sorry for backtracking here.

Reply:
Yes, I wasn't sure either. I will ask design what to do

Reply:
I will leave it as undefined, and not show the chart in case we have a mortality rate undefined

Reply:
Ok sounds good thanks!!